libr/debug/debug.c

```diff
@@ -343,6 +343,11 @@ R_API int r_debug_select(RDebug *dbg, int pid, int tid) {
 	if (dbg->h && dbg->h->select && !dbg->h->select (pid, tid))
 		return false;
 
+	{
+		char buf[32];
+		sprintf (buf, "pid %d", pid);
+		r_io_system (dbg->iob.io, buf);
```

Comment:
also i think static code analysis tools complain about using sprintf instead of snprintf